### PR TITLE
add cli

### PR DESCRIPTION
**Pull request by maxogden**
adds a CLI :D

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,10 +14,7 @@
   "bugs": {
     "url": "https://github.com/jonschlinkert/markdown-toc/issues"
   },
-  "license": {
-    "type": "MIT",
-    "url": "https://github.com/jonschlinkert/markdown-toc/blob/master/LICENSE"
-  },
+  "license": "",
```
Comment (jonschlinkert) [inline package.json]: missing a license name here

**Comment by jonschlinkert** [comment]
everything looks great. if you could fix the license field that would be awesome, then  I can merge. thanks a lot for doing this!

File: package.json (hunk 1 above)
Comment (maxogden) [inline package.json]: ah sorry, I just ran `npm init` to have npm fill in missing license details and I guess it didn't handle this. will fix

**Comment by maxogden** [comment]
@jonschlinkert done :)

File: package.json (hunk 1 above)
Comment (jonschlinkert) [inline package.json]: no worries, I figured. thanks!

**Comment by jonschlinkert** [comment]
bumped and published to npm. thanks again!

**Pull request merged**